posthog-react-native/src/posthog-rn.ts

```diff
@@ -35,12 +42,29 @@ export type PostHogOptions = PostHogCoreOptions & {
    * If you're already using the 'captureLifecycleEvents' options with 'withReactNativeNavigation' or 'PostHogProvider, you should not set this to true, otherwise you may see duplicated events.
    */
   captureNativeAppLifecycleEvents?: boolean
+
+  /**
+   * Enable Recording of Session Replays for Android and iOS
+   * Requires Record user sessions to be enabled in the PostHog Project Settings
+   * Experimental support
+   * Defaults to false
+   */
+  sessionReplay?: boolean
```

Comment:
Should we make this clearer that it's a boolean?
```suggestion
  enableSessionReplay?: boolean
```

Reply:
mm Android and iOS are just `sessionReplay` since its strongly typed languages, but happy to change it here.

Reply:
Nope, all good. Better to stay consistent. For context on the JS SDK it's `disbable_session_recording`